### PR TITLE
[ACTION / SOURCE] Github new actions #1430

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/github/sources/new-mention/new-mention.mjs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  ...common,
+  key: "github-new-mention",
+  name: "New Mention",
+  description: "Emit new events when you are @mentioned in a new commit, comment, issue or pull request",
+  version: "0.1.0",
+  type: "source",
+  hooks: {
+    async activate() {
+      const user = await this.github.getAuthenticatedUser();
+      this.db.set("login", user.login);
```
Comment (vellames-turing) [inline components/github/sources/new-mention/new-mention.mjs:13]: Can you create a setter function for `login`, please?

**Review by vellames-turing — CHANGES_REQUESTED** [review]
Hi there, just some minors

**Review by vellames-turing — APPROVED** [review]
Tks!

**Comment by lcaresia** [comment]
@dannyroosevelt 

For some reason `new-organization` source in some Github accounts don't do the polling in the organizations.

I don't found how to resolve this issue, what I do in this case?

[More context here](https://pipedream-users.slack.com/archives/C01E5KCTR16/p1655128700257509)

**Comment by dannyroosevelt** [comment]
> @dannyroosevelt
> 
> For some reason `new-organization` source in some Github accounts don't do the polling in the organizations.
> 
> I don't found how to resolve this issue, what I do in this case?
> 
> [More context here](https://pipedream-users.slack.com/archives/C01E5KCTR16/p1655128700257509)

Go ahead and skip that trigger for now

**Comment by vunguyenhung** [comment]
Hello @lcaresia, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[Github_1430.pdf](https://github.com/PipedreamHQ/pipedream/files/8968694/Github_1430.pdf)

**Comment by vunguyenhung** [comment]
Hello, per discussion with @lcaresia, I have tested again and all test cases are passed. Ready for release!

[Github_1430_2142.pdf](https://github.com/PipedreamHQ/pipedream/files/8976981/Github_1430_2142.pdf)

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**